testObj: bunny
visualize: True
handFile: /models/robotiq/urdf_openrave_conversion/robotiq_s.xml